src/store.js

```diff
@@ -15,10 +37,16 @@ export default new Vuex.Store({
   },
   getters: {
     colTemplate(state) {
-      return state.colArr.map(i => i["unit"]).join(" ");
+      // Maybe this mapping can be executed inside groupRepeatedUnits
+      // to avoid code repetition
```

Comment:
Yeah, I'd agree- I'll take this PR without that refactor, but since it's used both times this way, I'd add it to  `groupRepeatedUnits` 👍